lib/chef/resource/file/verification/json.rb

```diff
@@ -0,0 +1,52 @@
+#
+# Author:: Antony Thomas (<antonydeepak@gmail.com>)
+# Copyright:: Copyright (c) Chef Software Inc.
```

Comment:
This is your copyright now ours

Reply:
Not sure I understand. All files seem to have the same verbiage but for author.

Reply:
This is your work so it's not our copyright. If you'd like to copyright it with yourself or your company that's fine. If you want to leave it this way that's also fine.

Reply:
Thanks for pointing that out, I didn't pay attention to the copyright clause. I changed it to use my employers'